;qa

```diff
@@ -0,0 +1,171 @@
+package msgraph
```

Comment:
Is this file an accidental shell redirect? Could you delete this and rebase to remove it from history, thanks!

Reply:
Yes. It is an accidental file. Removed it in https://github.com/manicminer/hamilton/pull/179/commits/e4be3b4e541d11d14ca4a99185569f5e22731631